README.md

```diff
@@ -734,6 +759,43 @@ window.addEventListener("MSSDK:redeem-coupon-failed", evt =>
+
+```javascript
+{
+  checkoutReturnUrl: 'https://www.test.com', // The url to return the customer in case of a redirection during payment process in checkout component
+  myaccountReturnUrl: 'https://www.test.com', // The url to return the customer in case of a redirection during add/update payment detauls process in my account component
+  analytics: {
+      enabled: false, // Indicates if you're sending analytics data to Adyen. Default: true.
+  },
+  paymentMethodsConfiguration: {
+    card: {
+      name: 'name', //	String that is used to display the payment method name to the shopper.
+      billingAddressRequired: true, // Set to true to collect the shopper's billing address and mark the fields as required. Default: false
+      billingAddressMode: 'partial', // If billingAddressRequired is set to true, you can set this to partial to require the shopper's postal code instead of the full address. Default: 'full'
+      brands: ['visa'], // Array of card brands that will be recognized. For a list of possible values, refer to https://docs.adyen.com/payment-methods/cards/custom-card-integration#supported-card-types. Default: ['mc','visa','amex']
+      brandsConfiguration: { // Object where you can customize the icons for different brands.
+        visa: {
+            icon: 'https://...'
+        }
+      },
+      showBrandIcon: true, // Set to false to not show the brand logo when the card brand has been recognized. Default: true
+      showBrandsUnderCardNumber: true, // Shows brand logos under the card number field when the shopper selects the card payment method. Default: true
+      positionHolderNameOnTop: true, // Renders the cardholder name field at the top of the payment form. Default: false
+      styles: {}, // Set a style object to customize the card input fields. For a list of supported properties, refer to https://docs.adyen.com/payment-methods/cards/custom-card-integration#styling
+      billingAddressAllowedCountries: ['US', 'CA', 'BR', 'PL'], // Specify allowed country codes for the billing address. Default: The Country field dropdown menu shows a list of all countries.
+      minimumExpiryDate: null, // If a shopper enters a date that is earlier than specified here, they will see the following error: "Your card expires before check out date." Format: mm/yy
+      autoFocus: true // Automatically move the focus from date field to the CVC field. The focus also moves to the date field when the entered card number reaches the expected length. Default: true
+    },
+  },
+  locale: 'string', // The language used in the Drop-in UI. For possible values, see the https://docs.adyen.com/online-payments/web-drop-in/customization#supported-languages,
```

Comment:
I would add a more meaningful example here, like 'en-US'

Reply:
Yep. Forgot to replace it before commiting. Good eye :)